docs/fileformat-prose.md

```diff
@@ -148,14 +203,17 @@ DataFrames are saved as a columnar format in a group, so each column of a DataFr
 We save a little more information in the attributes here.
 
 ```python
->>> dict(f["obs"].attrs)
-{'_index': 'Cell',
```

Comment:
is the name of the Zarr group _index or Cell?

Reply:
The array coressponding to `adata.obs.index` is stored at `"obs/Cell"`, but I think this is documented adequately in the specification definition below.